action/pipeline/get_test.go

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) 2022 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package pipeline
+
+import (
+	"net/http/httptest"
+	"testing"
+
+	"github.com/go-vela/server/mock/server"
+
+	"github.com/go-vela/sdk-go/vela"
+)
+
+func TestPipeline_Config_Get(t *testing.T) {
+	// setup test server
+	s := httptest.NewServer(server.FakeHandler())
+
+	// create a vela client
+	client, err := vela.NewClient(s.URL, "vela", nil)
+	if err != nil {
+		t.Errorf("unable to create client: %v", err)
+	}
+
+	// setup tests
+	tests := []struct {
+		failure bool
+		config  *Config
+	}{
+		{
+			failure: false,
+			config: &Config{
+				Action:  "get",
+				Org:     "github",
+				Repo:    "octocat",
+				Page:    1,
+				PerPage: 10,
+				Output:  "",
+			},
+		},
+		{
+			failure: false,
+			config: &Config{
+				Action:  "get",
+				Org:     "github",
+				Repo:    "octocat",
+				Page:    1,
+				PerPage: 10,
+				Output:  "dump",
+			},
+		},
+		{
+			failure: false,
+			config: &Config{
+				Action:  "get",
+				Org:     "github",
+				Repo:    "octocat",
+				Page:    1,
+				PerPage: 10,
+				Output:  "json",
+			},
+		},
+		{
+			failure: false,
+			config: &Config{
+				Action:  "get",
+				Org:     "github",
+				Repo:    "octocat",
+				Page:    1,
+				PerPage: 10,
+				Output:  "spew",
+			},
+		},
+		{
+			failure: false,
+			config: &Config{
+				Action:  "get",
+				Org:     "github",
+				Repo:    "octocat",
+				Page:    1,
+				PerPage: 10,
+				Output:  "yaml",
+			},
+		},
+	}
+
+	// run tests
+	for _, test := range tests {
```

Comment:
Would you please name the tests and use `t.Run` to turn these into subtests?

Reply:
fixed 👍